src/stdatamodels/jwst/transforms/converters/jwst_models.py

```diff
@@ -17,15 +18,65 @@
     "MIRIWFSSDispersionConverter",
     "Rotation3DToGWAConverter",
     "GratingEquationConverter",
-    "V23ToSkyConverter",
     "SnellConverter",
     "CoordsConverter",
     "Rotation3DToGWAConverter",
     "Slit2MsaConverter",
     "Slit2GwaConverter",
+    "UnsupportedConverter",
+    "UnsupportedConverterError",
 ]
 
 
+class UnsupportedConverterError(Exception):
+    pass
+
+
+class UnsupportedConverter:
+    """
+    Class to catch legacy transforms that are no longer supported.
+
+    This converter contains tags for all converters that have been removed
+    from the JWST pipeline, but which may still be present in old files.
+    When encountered, an error will be raised and the WCS will not be deserialized.
+    """
+
+    types = []
+    tag_to_max_version = {"tag:stsci.edu:jwst_pipeline/v23tosky-*": "4.0.0"}
```

Comment:
```suggestion
    tag_to_max_version = {"tag:stsci.edu:jwst_pipeline/v23tosky-*": "4.1.0"}
```
Should this be 4.1.0?